.rubocop.yml

```diff
@@ -44,39 +58,14 @@ Style/Documentation:
 Style/DoubleNegation:
   Enabled: false
 
-Style/EmptyLinesAroundClassBody:
-  EnforcedStyle: empty_lines_except_namespace
-
-Style/EmptyLinesAroundModuleBody:
-  EnforcedStyle: empty_lines_except_namespace
-
 Style/FrozenStringLiteralComment:
   EnforcedStyle: never
 
 Style/HashSyntax:
   EnforcedStyle: ruby19
 
-# TODO: replace the [] below with default once a new version of
-#       rubocop comes out, since there is a PR for it:
-#       https://github.com/bbatsov/rubocop/pull/4038
-Style/PercentLiteralDelimiters:
```

Comment:
I've removed this, but I wasn't sure what the comment meant exactly. If only to replace square brackets `[]` or if it was meant as a placeholder for all of them. Anyway, the PR has been merged and I'll gladly apply any necessary changes.